### PR TITLE
Upgrading database management.

### PR DESCRIPTION
**Pull request by emanueljoivo**
Ps: Approve after #89. 

- [x] Change database code.
- [x] Refactor authentication and user management.
- [x] Refactor access database operations.
- [x]  Installation script.

### CODE REVIEW

File: src/main/java/org/fogbowcloud/app/core/datastore/managers/JobDBManager.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.fogbowcloud.app.core.datastore.repositories.JobRepository;
+import org.fogbowcloud.app.core.models.job.Job;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class JobDBManager {
+    private static JobDBManager instance;
+
+    @Autowired
```
Comment (wesleymonte) [inline src/main/java/org/fogbowcloud/app/core/datastore/managers/JobDBManager.java]: Is this notation required even using the set method?

File: src/main/java/org/fogbowcloud/app/core/models/command/Command.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    @Enumerated(EnumType.STRING)
+    private CommandState state;
+
+    @Column(name = EXIT_CODE_COLUMN_NAME)
+    private int exitCode;
+
+    public Command() {}
+
+    public Command(String command, CommandState state) {
```
Comment (wesleymonte) [inline src/main/java/org/fogbowcloud/app/core/models/command/Command.java]: Shouldn't this builder be private?

Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/core/models/command/Command.java]: Fixed in 9beb9b63b1b32e6bcb8c2bd1fcdfb05a6d03a48f.

File: src/main/java/org/fogbowcloud/app/core/datastore/managers/JobDBManager.java (hunk 1 above)
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/core/datastore/managers/JobDBManager.java]: Fixed in 8f304327bf8689f7bd7038b2332833c69e82f3bc.

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk8
 
 script:
-   - mvn clean install -B -V
+   - mvn clean install -DskipTests
```
[the remark this answers is not in the document]
Comment (emanueljoivo) [reply .travis.yml]: Fixed in 143965f6a64b6d8702e2fde1de465253937322d8.

File: src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolTaskSpec.java [hunk 4]
```diff
@@ -4,15 +4,15 @@
 
 public class ArrebolTaskSpec {
 
-    private String id;
+    private Long id;
```
[the remark this answers is not in the document]
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolTaskSpec.java]: Arrebol takes the task identifier as a parameter and now in Iguassu this identifier is a Long.

File: install.sh [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+sudo apt-get install -y nodejs
+
+install-iguassu() {
+	echo '--->  Installing Iguassu'
+	mvn clean install -DskipTests && cd ../
+}
+
+install-arrebol() {
+	echo '--->  Installing Arrebol'
+	wget https://github.com/ufcg-lsd/arrebol/archive/feature/remote-worker.zip
```
[the remark this answers is not in the document]
Comment (emanueljoivo) [reply install.sh]: Can u refactor this?

[the remark this answers is not in the document]
Comment (wesleymonte) [reply install.sh]: #93 added new script

**Pull request merged**